### ISSUE TITLE
How to best deal with copyto and pint?

### ISSUE THREAD

**Issue by NOhs** [hgrecco/pint#553]
The `copyto` function complains that `argument must be numpy.ndarray, not Quantity`. I guess that this numpy function is one of the few that is not supported by pint currently. My current approach to work around this is:

```python    
# ...
tmp = destination.magnitude
np.copyto(tmp, source.magnitude)
destination= tmp * source.units
```

Are there any cleaner options to use `copyto` with units?

**Comment by hgrecco** [comment]
Sadly there is no cleaner way to do this. It is not really that we have not supported, but that we cannot do it to the API that numpy exposes (See for example #232 )

**Comment by NOhs** [comment]
Too bad. But it's not a deal breaker for me. Pint still works great in most situations!

**Comment by hgrecco** [comment]
At the time, we did not find a way to make it cleanly. We wanted:

- NumPy Optional: Pint should work even without NumPy. This now might be possible as we have decoupled the registries providing different functionality.
- Interoperable: going from a subclass object to a wrapped object should be transparent
- Worth it: a large number of NumPy functions should become supported out of the box 

I am open to look into this again if there is a good proposal
